examples/des_y1_3x2pt/des_y1_3x2pt.yaml

```diff
@@ -1,13 +1,13 @@
 parameters:
   Omega_k: 0.0
   Omega_b: 0.0485
-  Omega_c: 0.2545
+  Omega_c: [0.2, 0.2545, 0.4]
   w0: -1.0
   wa: 0.0
   h: 0.682
-  sigma8: 0.801
+  sigma8: [0.6, 0.801,  0.9]
```

Comment:
Did you decide to punt a single sampled parameters discussion?

Reply:
Looking at this now.

Reply:
actually will punt on this for now, as it requires some thought about how things should look and a full rewrite